benchmarks/programs/test_kernel/src/main.rs

```diff
@@ -0,0 +1,21 @@
+#![cfg_attr(not(feature = "std"), no_main)]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+openvm_native_guest_macro::edsl_kernel! {
+    fn function_name(n: usize | Felt<F>) -> usize | Felt<F> {
+        compiler_output.txt
```

Comment:
we can address later, I would slightly prefer if this was a string with " " to match `include_bytes!`